app/src/androidTest/java/org/simple/clinic/overdue/AppointmentRepositoryAndroidTest.kt

```diff
@@ -1451,6 +1451,85 @@ class AppointmentRepositoryAndroidTest {
     assertThat(overdueAppointments).containsExactlyElementsIn(expectedAppointments)
   }
 
+  @Test
+  fun patients_without_phone_number_should_not_be_shown_when_fetching_overdue_appointments() {
+
+    val currentDate = LocalDate.parse("2018-01-05")
+
+    fun createRecord(
```

Comment:
`createPatientRecord`?

Reply:
Renamed to `createAppointmentRecord` in accordance with renaming the class to `RecordAppointment`.